app/components/parents/session-actions/SessionActionsSubmit.tsx

```
@@ -0,0 +1,59 @@
+  };
+
+  return (
+    <>
+      <div className="flex min-h-screen flex-col items-center justify-center pb-[88px] text-center">
+        <Result>
+          <Result.Image kind="document" />
+          <Result.Title>
+            <span dangerouslySetInnerHTML={{ __html: title }} />
+          </Result.Title>
+          <Result.Description>
+            <span dangerouslySetInnerHTML={{ __html: description }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)